### PR TITLE
get-to-fix-dojo

### PR DESCRIPTION
**Pull request by Quodss**
Fix incorrect Dojo output in `++get:to`

### CODE REVIEW

**Review by ripperi — APPROVED** [review]

File: content/language/hoon/reference/stdlib/2k.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ A cell of the last element in `a` along with the rest of queue `a`.
 ```
 > =s (~(gas to *(tree @)) `(list @)`~[1 2 3])
 > ~(get to s)
-[n=2 l={3} r={1}]
+[p=1 q={3 2}]
 
 > ~(get to ~)
 ! exit
```
Comment (ripperi) [inline content/language/hoon/reference/stdlib/2k.md:328]: `dojo: hoon expression failed`

Comment (Quodss) [reply content/language/hoon/reference/stdlib/2k.md:328]: @ripperi wdym? I copy the lines to Dojo and get the same output as in my PR

image

Comment (ripperi) [reply content/language/hoon/reference/stdlib/2k.md:328]: The failure case `~(get to ~)` output seems to be outdated as well. Might as well update that in the process.

Comment (Quodss) [reply content/language/hoon/reference/stdlib/2k.md]: sure, I made a commit

**Pull request merged**